packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -895,8 +969,13 @@ export const tagData = <
 		}, {}) as ReturnType<typeof tagData>;
 
 /**
- * Helper function to tag telemetry properties as CodeArtifacts. It supports properties of type
- * TelemetryBaseEventPropertyType as well as getters that return TelemetryBaseEventPropertyType.
+ * Helper function to tag telemetry properties as CodeArtifacts.
```

Comment:
Maybe add a "See also tagData" since I suggested a useful example to put in that guy's comment